docker-compose-ssl.yml

```diff
@@ -3,6 +3,7 @@ services:
   nginx-proxy:
     image: jwilder/nginx-proxy
     container_name: nginx-proxy
+    restart: always
```

Comment:
Why do we set to restart = always?

Reply:
For nginx-proxy we need it to always be running. When the host restarts for example we need it also to start automatically.

Reply:
This restarts the nginx-proxy upon start of the server or restart. This is set for all the containers.